src/users.js

```diff
@@ -1,20 +1,35 @@
 import { copyItem, getRandomIndex } from "./helper-functions";
 
+const getRandomUser = users => {
+  return users[getRandomIndex(users)]
+}
+
 const addUniqueRecipes = (userRecipes, newRecipe) => {
   if(!userRecipes.find(item => item.id === newRecipe.id)) userRecipes.push(newRecipe)
 };
 
-const addRecipeToCook = (user, recipe) => {
+const addRecipe = (userRecipes, newRecipe) => {
+  if (!userRecipes) userRecipes = [newRecipe];
```

Comment:
for readability, could userRecipes just be initialized to an empty array?
I know the next line avoids duplicates intrinsically but this way it'd be clearer

Reply:
done